### PR TITLE
Certificate renewal -- suggested edits

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: user/enterprise/platform-tips.md [hunk 1]
```diff
@@ -255,52 +255,55 @@ $ replicatedctl app start
...
-replicatedctl app stop
-sudo certbot renew
-replicatedctl app start
-```
-Set the executable permissions you'll need by doing `chmod +x /home/ubuntu/renew-certs.sh`.
+    replicatedctl app stop
+    sudo certbot renew
+    replicatedctl app start
+    ```
+    {: data-file="/home/ubuntu/renew-certs.sh"}
```
Comment (ggcampinho-travis) [inline user/enterprise/platform-tips.md]: Is this right? I don't know exactly what it means, but it seems to render regular text

Comment (philpeble) [reply user/enterprise/platform-tips.md]: Yes, this is what it looks like: https://deploy-preview-2270--musing-lamport-7fb66c.netlify.com/user/enterprise/platform-tips/#automate-renewal-of-your-lets-encrypt-ssl-certificate-with-a-cron-job

Comment (ggcampinho-travis) [reply user/enterprise/platform-tips.md]: Oh nice!

File: user/enterprise/platform-tips.md [hunk 2]
```diff
@@ -255,52 +255,55 @@ $ replicatedctl app start
...
-0 2 1 2,5,8,11 * /home/ubuntu/renew-certs.sh
-```
 
-**Note: Be aware that this process will also introduce downtime.**
+    ```
+    # Renews certs at 2am on the 1st of February, May, August, and November.
+    # Please change the configuration that applies to the certificate you are creating.
+    0 2 1 2,5,8,11 * /home/ubuntu/renew-certs.sh
+    ```
+
```
Comment (ggcampinho-travis) [inline user/enterprise/platform-tips.md]:
```suggestion
    {: data-file="/etc/crontab"}

```

**Pull request merged**